csharp/device/Microsoft.Azure.Devices.Client/HttpClientHelper.cs

```diff
@@ -123,7 +199,35 @@ sealed class HttpClientHelper : IHttpClientHelper
             return result;
 #endif
         }
+#endif
 
+#if NETMF
+        static HttpWebResponse ReadResponseMessageAsync(HttpWebResponse message)
+        {
+            // TODO
+            // unclear what is the purpose of this method and how to implement it
```

Comment:
that comment makes me a little worried I'll try and figure this out. thoughts welcome!